### PR TITLE
truncate begin #18

### PR DESCRIPTION
**Pull request by mapeveri**
| Q             | A
| ------------- | ---
| Is bugfix?    |❌
| New feature?  | ✔️
| Breaks BC?    |❌
| Tests pass?   | ✔️

PR related to issue #18

### CODE REVIEW

File: src/StringHelper.php [hunk 1]
```diff
@@ -430,4 +430,16 @@ public static function htmlspecialchars(string $string, int $flags, string $enco
             ? htmlspecialchars($string, $flags)
             : htmlspecialchars($string, $flags, $encoding ?: ini_get('default_charset'), $double_encode);
     }
+
+    /**
+     * Truncate the start string
```
Comment (samdark) [inline src/StringHelper.php]:
```suggestion
     * Truncate the string from the beginning
```

File: src/StringHelper.php [hunk 2]
```diff
@@ -430,4 +430,16 @@ public static function htmlspecialchars(string $string, int $flags, string $enco
...
+     *
+     * @param string $string string to process
+     * @param int $length total of character to process
```
Comment (samdark) [inline src/StringHelper.php]:
```suggestion
     * @param int $length total of character to truncate
```

**Review by samdark — APPROVED** [review]

**Comment by mapeveri** [comment]
@samdark I updated the PR :)

File: src/StringHelper.php [hunk 3]
```diff
@@ -430,4 +430,16 @@ public static function htmlspecialchars(string $string, int $flags, string $enco
...
     }
+
+    /**
+     * Truncate the string from the beginning
+     *
+     * @param string $string string to process
+     * @param int $length total of character to truncate
+     * @return string
+     */
+    public static function truncateBegin(string $string, int $length): string
```
Comment (cebe) [inline src/StringHelper.php]: Add a `, string $suffix = '…'` parameter like we have in the other truncate functions. Also would be good to move the function near to the other truncate functions.

File: tests/StringHelperTest.php [hunk 4]
```diff
@@ -401,4 +401,11 @@ public function testUcwords(string $string, string $expectedResult): void
     {
         $this->assertSame($expectedResult, StringHelper::ucwords($string));
     }
+
+    public function testTruncateBegin(): void
+    {
+        $this->assertSame('sms sent to …456.', 'sms sent to ' . StringHelper::truncateBegin('123456', 3) . '.');
```
Comment (xepozz) [inline tests/StringHelperTest.php]: I think it's 3 identical test cases. Is it right?
Remove the last dot from the strings and remove 2 last cases

File: src/StringHelper.php (hunk 3 above)
Comment (mapeveri) [reply src/StringHelper.php]: @cebe great idea :) 

I updated the PR

File: tests/StringHelperTest.php (hunk 4 above)
Comment (mapeveri) [reply tests/StringHelperTest.php]: @xepozz I removed the last 2 tests and add one more for test $suffix variable ¿it's ok?.

**Review by samdark — APPROVED** [review]

**Review by terabytesoftw — APPROVED** [review]

**Comment by samdark** [comment]
Thanks!

**Pull request merged**